kitsune/wiki/views.py

```diff
@@ -1244,7 +1244,7 @@ def helpful_vote(request, document_slug):
             # Send a survey if flag is enabled and vote wasn't helpful.
             if "helpful" not in request.POST:
                 survey = render_to_string(
-                    "wiki/includes/unhelpful_survey.html", {"vote_id": vote.id}
+                    "wiki/includes/unhelpful_survey.html", {"vote_id": vote.id}, request
```

Comment:
Without passing `request` into the `render_to_string`, it has been failing to render `{% csrf_token %}` in the `unhelpful_survey.html` template, since the `csrf_token` tag relies on a request context, so this fixes that issue.